src/libs/isAppInstalled/index.native.js

```diff
@@ -0,0 +1,15 @@
+import {Linking} from 'react-native';
+
+/**
+ * Gets the URLSchema and check the existence of that application
+ *
+ * @param {String} urlScheme
+ * @returns {Promise<Boolean>}
+ */
+export default async function isAppInstalled(urlScheme) {
+    try {
+        return await Linking.canOpenURL(`${urlScheme}://`);
```

Comment:
Could we replace await with Promise (for consistency throughout the codebase). See [this STYLE.md section](https://github.com/Expensify/Expensify.cash/blob/460f9a2040d1a4c4a14cb308557a9a50b50a0123/STYLE.md#esnext-are-we-allowed-to-use-insert-new-language-feature-why-or-why-not) for more info.

Reply:
Hi @Julesssss 
Added commit with converting async-await to a promise.
When calling the function developer can use then().catch() or async-await